tests/org.eclipse.passage.loc.billing.core.tests/OSGI-INF/l10n/bundle.properties

```diff
@@ -0,0 +1,23 @@
+#Properties file for org.eclipse.passage.loc.report.core.tests
+###############################################################################
+# Copyright (c) 2020 ArSysOp and others
+#
+# This program and the accompanying materials are made available under the
+# terms of the Eclipse Public License 2.0 which is available at
+# https://www.eclipse.org/legal/epl-2.0/.
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#     ArSysOp - initial API and implementation
```

Comment:
You can use your name and email here and in other places  - this is your contribution